moto/ec2/responses/elastic_network_interfaces.py

```diff
@@ -39,16 +40,43 @@ def delete_network_interface(self):
             return template.render()
 
     def describe_network_interface_attribute(self):
-        raise NotImplementedError(
-            "ElasticNetworkInterfaces(AmazonVPC).describe_network_interface_attribute is not yet implemented"
-        )
+        eni_id = self._get_param("NetworkInterfaceId")
+        attribute = self._get_param("Attribute")
+        if self.is_not_dryrun("DescribeNetworkInterfaceAttribute"):
+            eni = self.ec2_backend.get_all_network_interfaces([eni_id])[0]
+
+            if attribute == "description":
+                template = self.response_template(
+                    DESCRIBE_NETWORK_INTERFACE_ATTRIBUTE_RESPONSE_DESCRIPTION
+                )
+                return template.render(eni=eni)
```

Comment:
Let's move the return-statement to the bottom of the method, considering it is the same for all if/elif clauses